test/verify/files/mock-insights

```diff
@@ -138,9 +134,22 @@ class handler(BaseHTTPRequestHandler):
 
 
 def insights_server(port):
+    # Let's put the certs into /var/lib/insights so that SELinux
+    # allows insights-client to actually read ca.crt when running as a
+    # systemd service.
+    certdir = "/var/lib/insights/mock-certs"
+    if not os.path.exists(certdir):
+        os.makedirs(certdir)
+        subprocess.check_call(["sscg"], cwd=certdir)
```

Comment:
This might possibly require a `restorecon` call afterwards, to make them readable for insights?

Reply:
It doesn't seem to be necessary.